graphql/package.json

```diff
@@ -40,11 +45,17 @@
     "@babel/preset-env": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@types/jest": "^26.0.22",
+    "@typescript-eslint/eslint-plugin": "^6.4.0",
+    "@typescript-eslint/parser": "^6.11.0",
     "babel-jest": "^26.6.3",
+    "eslint-config-standard-with-typescript": "^39.1.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-promise": "^6.0.0",
     "nodemon": "^2.0.15",
-    "prettier": "^2.5.1",
+    "prettier": "^3.1.0",
     "ts-jest": "^29.0.3",
     "ts-node": "^8.3.0",
-    "typescript": "^4.8.4"
+    "typescript": "*"
```

Comment:
I searched locally for `: "*"` in package.json files under City-of-Helsinki's projects I have checked out:
```bash
City-of-Helsinki>$ ls -1d */
berth-federation-gateway//
berth-reservations-admin//
berth-reservations-ui//
berth-reservations//
bestpractice//
django-helusers//
django-munigeo//
django-orghierarchy//
events-helsinki-monorepo//
example-backend-profile//
hauki//
hcrc2//
hds2//
helsinki-design-system//
helsinki-profile-gdpr-api//
hobbies-helsinki//
kukkuu-admin//
kukkuu-ui//
kukkuu//
kuvaselaamo//
liikunta-helsinki//
linkedcomponents-ui//
linkedevents//
notification-service-api//
open-city-profile//
palvelutarjotin-admin//
palvelutarjotin-ui//
palvelutarjotin//
parkkihubi//
react-helsinki-headless-cms//
respa//
servicemap-ui//
servicemap//
smbackend//
structured-log-transfer//
tilavarauspalvelu-admin-ui//
tilavarauspalvelu-core//
tilavarauspalvelu-ui//
tunnistamo//
unified-search//
yjdh//
youth-federation-gateway//
youth-membership-admin-ui//
youth-membership-ui//
youth-membership//

City-of-Helsinki>$ grep -r --exclude-dir=node_modules ': "\*"' --include=package.json .
./tilavarauspalvelu-ui/admin-ui/package.json:    "common": "*",
./tilavarauspalvelu-ui/ui/package.json:    "common": "*",
./yjdh/frontend/benefit/applicant/package.json:    "@frontend/benefit-shared": "*",
./yjdh/frontend/benefit/applicant/package.json:    "@frontend/shared": "*",
./yjdh/frontend/benefit/handler/package.json:    "@frontend/benefit-shared": "*",
./yjdh/frontend/benefit/handler/package.json:    "@frontend/shared": "*",
./yjdh/frontend/benefit/shared/package.json:    "@frontend/shared": "*",
./yjdh/frontend/kesaseteli/employer/package.json:    "@frontend/kesaseteli-shared": "*",
./yjdh/frontend/kesaseteli/employer/package.json:    "@frontend/shared": "*",
./yjdh/frontend/kesaseteli/handler/package.json:    "@frontend/kesaseteli-shared": "*",
./yjdh/frontend/kesaseteli/handler/package.json:    "@frontend/shared": "*",
./yjdh/frontend/kesaseteli/shared/package.json:    "@frontend/shared": "*",
./yjdh/frontend/kesaseteli/youth/package.json:    "@frontend/kesaseteli-shared": "*",
./yjdh/frontend/kesaseteli/youth/package.json:    "@frontend/shared": "*",
./yjdh/frontend/tet/admin/package.json:    "@frontend/shared": "*",
./yjdh/frontend/tet/admin/package.json:    "@frontend/tet-shared": "*",
./yjdh/frontend/tet/shared/package.json:    "@frontend/shared": "*",
./yjdh/frontend/tet/youth/package.json:    "@frontend/shared": "*",
./yjdh/frontend/tet/youth/package.json:    "@frontend/tet-shared": "*",
```

So it looks like the `"*"` is used for shared/common components only in these City-of-Helsinki's projects.
It might be more transparent and normal to use `"typescript": "^5.2.2"` here IMHO. How do you see this?